common/src/main/resources/assets/controlling/lang/pl_pl.json

```diff
@@ -3,12 +3,14 @@
   "options.showConflicts": "Konfliktujące",
   "options.showNone": "Nieprzypisane",
   "options.availableKeys": "Dostępne klawisze",
-  "options.sort": "Sortow.",
+  "options.sort": "Sortuj",
   "options.category": "Kategoria",
   "options.key": "Klawisz",
```

Comment:
Both the `options.category` and `options.key` keys seem to be no longer in use anywhere in the code. It may be a good idea to remove them from the codebase one day to avoid confusing the translators and save them some work. But I leave this up to you, as this is your project.